clients/admin-ui/src/features/common/utils.ts

```diff
@@ -32,7 +32,7 @@ export const debounce = (fn: (props?: any) => void, ms = 0) => {
 };
 
 export const formatDate = (value: string | number | Date): string =>
-  format(new Date(value), "MMMM d, Y, KK:mm:ss z");
+  format(new Date(value), "MMMM d, y, KK:mm:ss z");
```

Comment:
this update was a breaking change in the new version of `date-fns`. It does NOT change our existing date formats.